src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ActiveConfiguredProjectsProvider.cs

```diff
@@ -127,14 +127,21 @@ public OrderPrecedenceImportCollection<IActiveConfiguredProjectsDimensionProvide
 
         public async Task<ActiveConfiguredObjects<ProjectConfiguration>?> GetActiveProjectConfigurationsAsync()
         {
-            ProjectConfiguration? activeSolutionConfiguration = _services.ActiveConfiguredProjectProvider.ActiveProjectConfiguration;
+            ProjectConfiguration? activeSolutionConfiguration = _services.ActiveConfiguredProjectProvider?.ActiveProjectConfiguration;
 
             if (activeSolutionConfiguration == null)
             {
                 return null;
             }
 
-            IImmutableSet<ProjectConfiguration> configurations = await _services.ProjectConfigurationsService.GetKnownProjectConfigurationsAsync();
+            Task<IImmutableSet<ProjectConfiguration>>? getConfigurationsTask = _services.ProjectConfigurationsService?.GetKnownProjectConfigurationsAsync();
+
+            if (getConfigurationsTask == null)
+            {
+                return null;
```

Comment:
Behaviour change. Returns null rather than NRE.

Reply:
! this on ProjectConfigurationsService - this will never occur.